docs/github_actions.rst

```diff
@@ -374,6 +377,23 @@ in our demo repository.
 
 .. code-block:: yaml
 
+   name: Beta Package and Install
+
+   on:
+      push:
+         branches:
+            - master
+         paths-ignore:
+            - 'docs/**'
+            - 'README.md'
+
+   env:
+      CUMULUSCI_KEYCHAIN_CLASS: cumulusci.core.keychain.EnvironmentProjectKeychain
```

Comment:
This is no longer needed as of merge that took place earlier today.
```suggestion
```

Reply:
@Br4nd0R thanks for the change and the merge :)